followthemoney/schema.py

```diff
@@ -129,9 +132,12 @@ def __init__(self, model: "Model", name: str, data: SchemaSpec) -> None:
         #: inheritance.
         self.abstract = as_bool(data.get("abstract"), False)
 
+        #: This schema is deprecated and should not be used.
+        self.deprecated = as_bool(data.get("deprecated", False))
+
         #: Hide this schema in listings.
         self.hidden = as_bool(data.get("hidden"), False)
-        self.hidden = self.hidden and not self.abstract
+        self.hidden = self.hidden and not self.abstract and not self.deprecated
```

Comment:
This would make any deprecated schema visible, even if it was hidden previously?

Reply:
Removed.